libs/pageserver_api/src/reltag.rs

```diff
@@ -108,9 +108,22 @@ impl RelTag {
 /// These files are divided into segments, which are divided into
 /// pages of the same BLCKSZ as used for relation files.
 ///
-#[derive(Debug, Clone, Copy, Hash, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    strum_macros::FromRepr,
```

Comment:
If we're going to use from_repr() now, let's change the existing match-statements in pgdatadir_mapping.rs to do that too.

Is there corresponding to_repr() function? Or how does one get the integer representation?

Reply:
Just wan to notice that it is not the first time when `from_repr` used in Neon.
Another place is pageserver/src/metrics.rs (from which I actually grab this code).
`to_repr` is not needed because it is possible to convert enum to number using standard cast operator: `as u32`.

Refactoring pgdatadir_mapping.rs  in `strum_macros::FromRepr`definitly makes sense, but I almost sure that you will say that it should be done in separate PR.